rpcs3/rpcs3qt/game_list_frame.cpp

```diff
@@ -2226,6 +2220,13 @@ void game_list_frame::FocusAndSelectFirstEntryIfNoneIs()
 
 void game_list_frame::closeEvent(QCloseEvent *event)
 {
+	WaitAndAbortSizeCalcThreads();
+	WaitAndAbortRepaintThreads();
+	gui::utils::stop_future_watcher(m_parsing_watcher, true);
+	gui::utils::stop_future_watcher(m_refresh_watcher, true);
+
+	SaveSettings();
+
```

Comment:
There seems to be a fundamental misunderstanding of the QDockWidget::closeEvent...
This event is only triggered if you manually click the X in the dock.
So this change is wrong.

Reply:
main_windoww::closeEvent also calls m_game_list_frame->SaveSettings().

Reply:
Removed it.

Reply:
> main_windoww::closeEvent also calls m_game_list_frame->SaveSettings().

Well. now it probably isn't saved when you actually use the closeEvent.
I don't understand the constant need to change everything seemingly just out of boredom